src/controllers/applicant.ts

```diff
@@ -43,29 +44,63 @@ export const createApplicant = async (req: Request, res: Response): Promise<void
     }
 };
 
-export const updateApplicantStatus = async (req: Request, res: Response): Promise<void> => {
-    try {
-        const valid_status = ['Pending', 'Accepted', 'Rejected', 'Scheduled'];
-        const my_applicant = await Applicant.findOne({ _id: req.body.id });
+export const updateApplicantFields = async (req: Request, res: Response): Promise<void> => {
+    const valid_status = [
+        'Pending',
+        'Screened: Accepted',
+        'Screened: Rejected',
+        'Scheduled for Interview',
+        'Final Decision: Accepted',
+        'Final Decision: Rejected',
+        'Archived: Rejected',
```

Comment:
Sorry - one more small change, can we added a "Interviewed" and "Screened" state here too so we don't have to move directly from scheduled for interview to a decision (and same for after screening an applicant)

Reply:
I've added those states now in my most recent commit